packages/react-ui/components/GlobalLoader/useGlobalLoaderViewParams.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+
+import { usePrevious } from './usePrevious';
+import { GlobalLoaderViewProps } from './GlobalLoaderView';
+
+/**
+ * Вычисляет необходимые параметры Глобального Лоадера
+ *
+ * @param status - статус Глобального Лоадера: 'success' | 'error' | 'standard' | 'accept'
+ * @param ref - ref Global Loader View
+ * @returns - возвращает ширину, левое положение и ширину полоски ГЛ при переходе в состояние ошибки
+ */
```

Comment:
Лучше использовать названия компонентов на английском и писать их слитно в обратных кавычках (так они будут приятно подсвечиваться `IDE`):
```suggestion
/**
 * Вычисляет необходимые параметры `GlobalLoader`
 *
 * @param status - статус `GlobalLoader`
 * @param ref - ref `GlobalLoaderView`
 * @returns - возвращает ширину, левое положение и ширину полоски `GlobalLoader` при переходе в состояние ошибки
 */
```

Также _"Вычисляет необходимые параметры"_ звучит слишком глобально (_намеренный каламбур_), хук всё-таки вычисляет размер полоски в зависимости от статуса. Нужно это как-то обыграть.

Reply:
Следовательно и название самого хука стоит изменить